Doc/library/inspect.rst

```diff
@@ -401,6 +401,11 @@ attributes:
    Return ``True`` if the object is a built-in function or a bound built-in method.
 
 
+.. function:: ismethodwrapper(object)
+
+   Return ``True`` if the object is a method-wrapper.
```

Comment:
Since method wrappers are a rather esoteric concept, this should include more details and/or a reference to more information about them.

Reply:
I tried to add more details, Can you check again

Reply:
I suggest adding a second line, something like:

```restructuredText
These are instances of :class:`~types.MethodWrapperType`, such as :meth:`~object.__get__`.
```

Reply:
I have followed your suggestions, but I think [datamodel](https://docs.python.org/3/reference/datamodel.html) does not have things like `objects() .__ str__ ` at this address, do I need to write them in this pr into the file?

Reply:
**There is no redirect.**

![Capture](https://user-images.githubusercontent.com/19157033/94506887-20a32c00-0217-11eb-9207-94515b58d087.PNG)